swan-lake/learn-the-language/distinctive-language-features/advanced-general-purpose-language-features.md

```diff
@@ -666,6 +666,102 @@ int cp = ch.toCodePointInt();
 
 In the above code example, **``ch``** is of the ``string:Char`` type. You can also extract the code point of the character by calling the **``toCodePointInt()``** function. This ``string:Char`` subtype is analogous to the XML subtypes like XML elements or text.
 
+## Regular Expressions
+
+Ballerina provides first-class support for regular expressions, which are represented by the ``RegExp`` built-in type defined in the ``lang.regexp`` module. You can use the alias ``string`` to create a variable for regular expressions, even though it is defined in the ``lang.regexp`` module.
```

Comment:
```suggestion
Ballerina provides first-class support for regular expressions, represented by the built-in ``RegExp`` type defined in the ``lang.regexp`` module. Additionally, the ``lang.string`` module has a type alias named the same so that one can use either ``regexp:RegExp`` or  ``string:RegExp` to create variables for regular expressions.
```